src/server/search_family.h

```diff
@@ -22,9 +22,19 @@ class SearchFamily {
  public:
   static void Register(CommandRegistry* registry);
 
+  struct IndexData {
+    enum DataType { HASH, JSON };
+
+    // Get numeric OBJ_ code
+    uint8_t GetObjCode() const;
```

Comment:
I just didn't want to include redis headers here, so that's why I didn't set enum values

Reply:
I think that these types are not even `enum`s in our redis headers anyway :)